SignUpActivity.java

```diff
@@ -0,0 +1,137 @@
+package com.manoj.androidclub;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import androidx.appcompat.app.AppCompatActivity;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import java.util.Calendar;
+
+public class SignUpActivity extends AppCompatActivity {
+
+    DatabaseReference mDatabase;
+    FirebaseUser mCurrentUser;
+    String mEmail;
+    String mName;
+    String mDepartmant;
+    String mYear;
+    String mRegNumber;
+    String mCollege;
+    String mPassword;
+    String mConfirmPassword;
+
+    private EditText EmailEditText;
+    private EditText NameEditText;
+    private EditText DepartmantEditText;
+    private EditText YearEditText;
+    private EditText CollegeEditText;
+    private EditText RegNumberEditText;
+    private EditText ClassEditText;
+    private EditText PasswordEditText;
+    private EditText ConfirmPasswordEditText;
+    private Button SubmitButton;
```

Comment:
it should be named as mEmailEditText and so on , anyways no issues for now 
Great work

Reply:
Ok I will resolve this problem in next pull